File: Courses/JavaScript Part1/Conditional Statements - lecture 6/Javascript Sources/FindBiggestInteger.js
/*
 * Write a script that finds the biggest of three integers using nested if statements.
 */

(function() {
    var firstNumber = 3;
    var secondNumber = 7;
    var thirdNumber = 2;

    console.log("The first number is:" + firstNumber);
    console.log("The second number is:" + secondNumber);
    console.log("The third number is:" + thirdNumber);

    if (firstNumber > secondNumber && firstNumber > thirdNumber) {
        console.log("The first number " + firstNumber + " is the biggest one.");
    }
    else if (secondNumber > firstNumber && secondNumber > thirdNumber) {
        console.log("The second number " + secondNumber + " is the biggest one.");
    }
    else if (thirdNumber > firstNumber && thirdNumber > secondNumber) {
        console.log("The third number " + thirdNumber + " is the biggest one.");
    }
    else if (firstNumber === secondNumber || secondNumber === thirdNumber || firstNumber === thirdNumber) {
        console.log("At least two of the numbers are equal.");
    }
})();

File: Courses/C# Part 1/ConditionalStatements - lecture 5/3.FindBiggestInteger/FindBiggestInteger.cs
using System;

/*
 * Write a program that finds the biggest of three integers using nested if statements.
 */

class FindBiggestInteger
{
    static void Main()
    {
        int firstNumber;
        int secondNumber;
        int thirdNumber;
        bool isFirstInt, isSecondInt, isThirdInt;
        Console.WriteLine("Please enter the first number");
        isFirstInt = int.TryParse(Console.ReadLine(), out firstNumber);
        Console.WriteLine("Please enter the second number");
        isSecondInt = int.TryParse(Console.ReadLine(), out secondNumber);
        Console.WriteLine("Please enter the third number");
        isThirdInt = int.TryParse(Console.ReadLine(), out thirdNumber);
        if (!isFirstInt||!isSecondInt||!isThirdInt)
        {
             Console.WriteLine("Invalid input detected.");
             Console.WriteLine("Press any key to restart");
             Console.ReadKey(true);
             Console.Clear();
             Main();
        }
        if(firstNumber > secondNumber && firstNumber > thirdNumber)
        {
            Console.WriteLine("The first number {0} is the biggest one.", firstNumber);
        }
        else if (secondNumber > firstNumber && secondNumber > thirdNumber)
        {
            Console.WriteLine("The second number {0} is the biggest one.", secondNumber);
        }
        else if (thirdNumber > firstNumber && thirdNumber > secondNumber)
        {
            Console.WriteLine("The third number {0} is the biggest one.", thirdNumber);
        }
        else if (firstNumber==secondNumber || secondNumber==thirdNumber || firstNumber == thirdNumber)
        {
            Console.WriteLine("At least two of the numbers are equal.");
        }
        Console.WriteLine("Press any key to restart");
        Console.ReadKey(true);
        Console.Clear();
        Main();
    }
}
